galvan-support/src/main/java/org/terracotta/testing/config/StartupBuilder.java

```diff
@@ -22,8 +22,9 @@
 
 public interface StartupBuilder {
   StartupBuilder tcConfig(Path tcConfig);
-  StartupBuilder serverInstallationDir(Path serverInstallationDir);
-  StartupBuilder testParentDir(Path testParentDir);
+  StartupBuilder serverKitDir(Path serverInstallationDir);
```

Comment:
Variable name can match the API to avoid naming confusion. Also, `kitDir()` should suffice for the API name as the kit directory is the same for the whole test suite.